packages/@ama-sdk/showcase-sdk/openapitools.json

```diff
@@ -3,6 +3,14 @@
   "spaces": 2,
   "generator-cli": {
     "version": "6.6.0",
-    "storageDir": ".openapi-generator"
+    "storageDir": ".openapi-generator",
+    "generators": {
+      "ama-sdk-showcase-sdk": {
+        "generatorName": "typescriptFetch",
+        "output": ".",
+        "inputSpec": "./swagger-spec.yaml",
+        "globalProperty": "stringifyDate"
```

Comment:
maybe we can put an object it will be easier to read if we have a 2nd property to add at some point